R/shiny-getters.R

```diff
@@ -79,31 +80,29 @@
 #'     [addEventListener()](https://github.com/vega/vega/tree/master/packages/vega-view#view_addEventListener)
 #' @export
 #'
-vw_shiny_get_signal <- function(outputId, name, body_value = "value") {
+vw_shiny_get_signal <-
+  function(outputId, name, body_value = "value", id = NULL) {
```

Comment:
If you are going over 80 spaces, our convention is to put the overflowing arguments on the following lines, e.g.:

```r
vw_shiny_get_signal <- function(outputId, name, body_value = "value", 
                                moduleId = NULL) {
}
```